zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4441,6 +4463,63 @@ components:
         email:
           description: The email of the user.
           type: "string"
+    CamundaUser:
+      type: "object"
+      properties:
+          userId:
+            description: The ID of the user.
```

Comment:
Does this have to be exposed in the API? Would this ever be something other than the userKey?

Reply:
`userId` as far as I can make out from the existing webapps is actually the `username` for example `"userId": "demo",`. I think its something the respective webapps team can answer but for now I've gone with caution and added it for consistency